Tests/AppKernel.php

```diff
@@ -36,33 +36,28 @@ public function registerBundles() : iterable {
 		];
 	}
 
-	public function registerContainerConfiguration(LoaderInterface $loader) {
+	public function registerContainerConfiguration(LoaderInterface $loader): void
+	{
 		$loader->load($this->configFile);
 	}
 
-	public function getCacheDir() : string {
-		if (array_key_exists('CACHE_DIR', $_ENV)) {
-			return $_ENV['CACHE_DIR'] . DIRECTORY_SEPARATOR . $this->environment;
-		}
-
-		return parent::getCacheDir();
+	public function getCacheDir(): string
+	{
+		return sprintf('%scache', $this->getBaseDir());
 	}
 
-	public function getLogDir() : string {
-		if (array_key_exists('LOG_DIR', $_ENV)) {
-			return $_ENV['LOG_DIR'] . DIRECTORY_SEPARATOR . $this->environment;
-		}
-
-		return parent::getLogDir();
+	public function getLogDir(): string
+	{
+		return sprintf('%slog', $this->getBaseDir());
 	}
 
-	public function serialize() {
-		return serialize([$this->environment, $this->configFile]);
+	public function getProjectDir(): string
+	{
+		return __DIR__;
 	}
 
-	public function unserialize($data) {
-		list($environment, $configFile) = unserialize($data);
-		$this->__construct($environment, $configFile);
+	private function getBaseDir(): string
+	{
+		return sprintf('%s/craue-form-flow-bundle/var/', sys_get_temp_dir());
```

Comment:
What's the reason for this change? Was it meant to simplify things?
In fact, I still need these env vars while running tests locally to put temporary files on a RAM drive. 🤓

Reply:
Yeah, I think it was to remove the other methods, so if you need the env vars, keep them